### PR TITLE
Allow ci-plutus-benchmark.sh to be run locally

### PR DESCRIPTION
**Pull request by zeme-iohk**

### CODE REVIEW

File: scripts/ci-plutus-benchmark.sh [hunk 1]
```diff
@@ -78,4 +76,9 @@ echo -e "</details>"
 jq -Rs '.' bench-compare-result.log >bench-compare.json
 
 echo "[ci-plutus-benchmark]: Posting results to GitHub ..."
-curl -s -H "Authorization: token $(</run/keys/buildkite-github-token)" -X POST -d "{\"body\": $(<bench-compare.json)}" "https://api.github.com/repos/input-output-hk/plutus/issues/${PR_NUMBER}/comments"
\ No newline at end of file
+if [ -v GITHUB_TOKEN ] ; then
+   echo "[ci-plutus-benchmark]: Posting results to GitHub ..."
```
Comment (michaelpj) [inline scripts/ci-plutus-benchmark.sh:80]: double echo

Comment (zeme-iohk) [reply scripts/ci-plutus-benchmark.sh:80]: will fix

**Comment by thealmarty** [comment]
Thank you!

**Comment by thealmarty** [comment]
maybe worth adding instructions to [here](https://github.com/input-output-hk/plutus/blob/master/CONTRIBUTING.adoc#benchmarking-prs) as well?

**Pull request merged**